common/changes/@microsoft/api-extractor/setter-fix_2022-05-17-23-58.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@microsoft/api-extractor",
+      "comment": "Fix bug in how API properties are processed introduced by https://github.com/microsoft/rushstack/pull/3396",
```

Comment:
@iclanton I tuned up this change file in 360f72a141c6af4df7485d0b4a9f716f18a7bd2f:

`Fix a recent regression that produced an error "Cannot read properties of undefined" (GitHub #3423)`

It seems better to (1) reference the issue rather than shaming the PR 😋 and (2) give a little more detail about the error to help someone who is searching for a fix.